pkg/capture/capturetypes/config.go

```diff
@@ -0,0 +1,41 @@
+package capturetypes
+
+// IfaceChange denotes the result from an interface / config update of an interface
+type IfaceChange struct {
+	Name    string
+	Success bool
+}
+
+// IfaceChanges denotes a list of IfaceChange instances
+type IfaceChanges []IfaceChange
```

Comment:
What's your take on `[]IfaceChange` vs. `[]*IfaceChange`? I usually gravitate to the latter, but cannot put my finger down why exactly.

Reply:
And because I have a mild fixation with sorting, please add a `(IfaceChanges).SortByName` method.

Reply:
As for the `[]*IfaceChange`, I wouldn't do that. Go best practices recommend to avoid unnecessary points for simple structs (which this clearly qualifies as), in particular in slices (because that's already a referenced type). On top of that I don't see any benefit while adding dangers (such as unwanted in-place manipulation and / or panics because of some `nil` pointer inside the slice).

As for the sorting, sure, but let's stick with the implementation of a `Less()` method, that way it satisfies a standard interface for sorting and you can use Go intrinsic to do so.

Reply:
Makes perfect sense in this case then, since we really just return "read-only" data.

As for Less(), good idea! I would usually resort to implementing a custom method, but this is more elegant and flexible.

Reply:
Done.